packages/webawesome/src/components/details/details.test.ts

```diff
@@ -24,6 +24,13 @@ describe('<wa-details>', () => {
         });
       });
 
+      it('should reflect the name property', async () => {
+        const el = await fixture<WaDetails>(html`<wa-details></wa-details>`);
+        el.name = 'test'
+        await aTimeout(100);
```

Comment:
This should work instead of an arbitrary delay

```suggestion
        await el.updateComplete;
```